ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -76,15 +76,15 @@ Authorize rhods-dashboard service account
 Login To RHODS Dashboard
    [Arguments]  ${ocp_user_name}  ${ocp_user_pw}  ${ocp_user_auth_type}
    # Wait until we are in the OpenShift auth page or already in Dashboard
-   ${expected_text_list}=    Create List    Log in with
+   ${expected_text_list}=    Create List    Log in with    Data Science Projects
```

Comment:
hm probably this whole kw needs a review, including expected behavior.
This "data science projects", refers to the one in the left menu or it means the user is already in the "Data science projects" page? Does it make sense to expect the user to see "data science projects" at login step? maybe yes if the login is not required (expected?)

Reply:
I did not change the logic of this KW, I reverted it back to the original behavior (that was there before my previous PR):
`Create List    Log in with    Data Science Projects`

I don't mind changing it, and it actually works without "Data Science Projects", but keeping "Data Science Projects":
1) Does not affect performance.
2) It might be required when opening the Dashboard with an active login session, where the "Log in with" message does not exist.